tests/test_check_fasta_insdc.py

```diff
@@ -102,3 +103,9 @@ def test_assess_fasta_matches_metadata_integration(self):
             'associated_analyses': ['analysis'],
             'assembly_in_metadata': 'GCA_000146045.2'
         }
+
+    @pytest.mark.integration
+    def test_get_containing_assemblies(self):
+        results = get_containing_assemblies('6ac8f815bf8e845bb3031b73f812c012')
+        assert len(results) == 14
+        assert 'GCA_000001405.29' in results
```

Comment:
This test doesn't run on Github, but if you run it locally note that it will pass only inconsistently until EBIvariation/contig-alias#129 is deployed. This is due to inconsistent ordering of the results, which causes the pagination to sometimes miss data.

Reply:
Created [EVA-3830](https://embl.atlassian.net/browse/EVA-3830)